lib/models/wallet_connect.dart

```diff
@@ -0,0 +1,25 @@
+/// WalletConnect Methods
+abstract class WcMethods {
+  /// Get the client to request a list of all accounts in wallet
+  static const wRequestAccounts = "wallet_requestAccounts";
+
```

Comment:
can this be renamed as "qubic_getAccounts"
equivalent to what some other networks have (i.e "solana_getAccounts")